### ISSUE TITLE
Cannot "open" a project by the folder

### ISSUE THREAD

**Issue by shiffman** [processing/p5.js-editor#139]
While I now can drag a folder onto the dock (#131), when I try to open something by FILE --> OPEN, selecting a directory requires me to then select a file inside a directory.  I would prefer to open the project directly from the folder itself.  I think?

**Comment by antiboredom** [comment]
Agreed that this would be much better. Ideally you would be able to select a folder or a file, and both would work. But nodewebkit doesn't seem to let you do this, or didn't last time I tried. I'll take another look though since the project has evolved a bit since I did the basic open file/project dialog.
